classes/use-imx-headers.bbclass

```diff
@@ -0,0 +1,19 @@
+# Easy use of linux-imx-headers by packages
+#
+# This allow to easy reuse of binary packages among similar SoCs. The
+# usual use for this is to share SoC specific packages among different
+# boards independently of the kernel version it is using, as far it is
+# ABI compatible with the official version it will just work.
+#
+# All recipes using this class ought to depend on linux-imx-headers
+# and by default to use MACHINE_SOCARCH.
+#
+# Please use the STAGING_IMX_INCDIR variable to refer to the installed
+# headers.
+#
+# Copyright 2018 (C) O.S. Systems Software LTDA.
+
+DEPENDS_imx_append = " linux-imx-headers"
+PACKAGE_ARCH_imx ?= "${MACHINE_SOCARCH}"
+
+STAGING_IMX_INCDIR = "${STAGING_INCDIR}/imx"
```

Comment:
STAGING_INCDIR_IMX seems a better name since the word order follows the directory order.

Reply:
Great, mind to fix it?

Reply:
Will do.